Neos.Fusion/Classes/View/FusionView.php

```diff
@@ -46,7 +47,8 @@ class FusionView extends AbstractView
         'fusionGlobals' => [null, 'Additional global variables; merged together with the "request". Must only be specified at creation.', FusionGlobals::class],
         'packageKey' => [null, 'The package key where the Fusion should be loaded from. If not given, is automatically derived from the current request.', 'string'],
         'debugMode' => [false, 'Flag to enable debug mode of the Fusion runtime explicitly (overriding the global setting).', 'boolean'],
-        'enableContentCache' => [false, 'Flag to enable content caching inside Fusion (overriding the global setting).', 'boolean']
+        'enableContentCache' => [false, 'Flag to enable content caching inside Fusion (overriding the global setting).', 'boolean'],
+        'renderHttpResponse' => [false, 'Flag to render fusion as http repose for advanced form support and Neos.Fusion:Http.ResponseHead support.', 'boolean'],
```

Comment:
The more i think about it, the more i want to invert that flag and make it breaking.
Our tests could easily set that flag to render the mixed output directly.

My reasoning for this is that the current Fusion FusionView behaves incorrect.
The ViewInterface allows to return `string|ActionResponse|ResponseInterface|StreamInterface|\Stringable`
But we violate this by returning the raw runtime value instead, which is `mixed` as it could contain raw arrays or non string-able objects or integers...

By introducing a flag like `returnMixed` or `unsafeRawMode` we empathise that this is not the correct way and it violates the interface.

Ideally people using the FusionView should not notice this change as returning a `ResponseInterface` is okay by definition and must (is) accounted by the controllers.

As a migration people could either use the propsed unsafe rawmode, or call `getContents` on the response to get a string.